_board/swan.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: download
+board_id: "swan"
+title: "Swan Download"
+name: "Swan"
+manufacturer: "Blues Wireless"
+board_url: "https://blues.io/products/swan"
+board_image: "swan.jpg"
+date_added: 2021-9-29
+features:
+  - Feather-Compatible
+  - Battery Charging
```

Comment:
not sure about "Battery Charging" and "Breadboard-Friendly" as features. not sure how prominent these are on the listing (or if this is for SEO), but maybe "55 GPIO" and "Three independent power options" would be better?

Reply:
These are limited by Adafruit in terms of what I can select. 🤷